db/migrate/20231026033443_create_comment.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComment < ActiveRecord::Migration[7.1]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :users,foreign_key:true
+      t.references :posts,foreign_key:true
```

Comment:
- [ ] The foreign keys between users table and comments table is `user_id`, not `users_id` and posts table and comments table is `post_id`, not `posts_id`. Please fix that.

[hint] I will be like that:

```ruby
t.references :user, foreign_key: true
t.references :post, foreign_key: true
```